src/main/java/dk/bec/bookanything/controller/FacilityTypeController.java

```diff
@@ -0,0 +1,74 @@
+package dk.bec.bookanything.controller;
+
+
+import dk.bec.bookanything.dto.FacilityTypeDto;
+import dk.bec.bookanything.mapper.FacilityTypeMapper;
+import dk.bec.bookanything.model.FacilityTypeEntity;
+import dk.bec.bookanything.service.FacilityTypeService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.List;
+
+@RestController
+@RequestMapping("/API")
+public class FacilityTypeController {
+
+    private final FacilityTypeService facilityTypeService;
+    private final FacilityTypeMapper facilityTypeMapper;
+
+    @Autowired
```

Comment:
`@Autowired` annotation on constructor is not needed when using spring >5. Moreover, since I saw that you already used lombok library, try to use `@RequiredArgsConstructor` on class level and provide all fields with final keyword. It should create constructor on runtime, so you do not need to declare this explicitly in code